README.md

```diff
@@ -76,29 +106,36 @@ We also provide wrappers for the Luxonis cameras which we use internally. They a
 # Installation
 
 ```
-Note: This package has only been tested on Ubuntu 22.04.
+Note: This package has been tested on Ubuntu 22.04 and macOS (with M1 Pro processor).
 ```
+## Git LFS
+This repository uses Git LFS to store large files. You need to install it before cloning the repository.
 
-Install everything in "production" mode:
+### Ubuntu
 ```console
-pip install pollen_vision[all]
+sudo apt-get install git-lfs
 ```
 
-OR Install only the modules you want: 
+### macOS
 ```console
-pip install pollen_vision[depthai_wrapper]
-pip install pollen_vision[vision]
+brew install git-lfs
 ```
 
-## Dev mode
+## Install from source
```

Comment:
Maybe add another section to install without cloning the repo with the cmd

`pip install "pollen-vision[vision] @ git+https://github.com/pollen-robotics/pollen-vision.git@main"` ?